Implement `b2 file unhide` command (#297)
reef-technologies/B2_Command_Line_Tool: changelog.d/+b2_file_hide.changed.md

> `b2 file hide` supports both the new `b2_uri` and the old `bucket_name file_name` parameters syntax, but the --help text only mentions the new syntax.

Reviewer: ```suggestion
`b2 file hide` supports both the new `b2_uri` and deprecated `bucket_name file_name` parameters syntax.
```

> b2v3 hide-file remains unchanged and only supports the `bucket_name file_name` syntax.

Reviewer: ```suggestion
```
